src/main/java/com/zontik/groshiky/repository/AccountRepository.java

```diff
@@ -7,4 +7,5 @@
 @Repository
 public interface AccountRepository extends JpaRepository<Account, Integer> {
       Account findByName(String name);
+      Account findAllById(Integer id);
```

Comment:
Probably you meant findById. It doesn't make sense to have "all" accounts with the same id :)